packages/ui/locales/en.json

```diff
@@ -1,4 +1,5 @@
 {
+    "transactions": "transactions",
```

Comment:
Add this key to their corresponding component and let's try to follow a convention like:

```
{
  [componentName]: { 
    [key]: "translation"
  }
}
```

Reply:
The locales are not filled yet. This is just an example to prove it works.